src/tablo/plugins/columnsReorder/columnsReorderGrid.js

```diff
@@ -62,8 +62,7 @@ export default (Grid) =>
         const target_index = list.indexOf(target);
         if (source_index <= target_index) {
           return [...list.slice(0, source_index), ...list.slice(source_index + 1, target_index + 1), source, ...list.slice(target_index + 1)];
-        }
-        if (target_index <= source_index) {
+        } else {
```

Comment:
this has changed. to reflect previous logic you should keep the `if` and add a `return undefined` at the bottom of the function

/cc @francescogior